src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -5594,6 +5594,96 @@ class TestClass { }";
             N(SyntaxKind.EndOfFileToken);
         }
 
+        [Fact]
+        public void RefExtensionMethodsNotSupportedBefore7_2_RefReadOnlySyntax()
```

Comment:
If you are really interested in testing result of GetParseDiagnostics, simply add the checks in the tests in RefExtensionMethodsTests.cs

---
In reply to: [123673035](https://github.com/dotnet/roslyn/pull/20403#discussion_r123673035) [](ancestors = 123673035)